test-api-js/todo.test.js

```diff
@@ -22,6 +22,29 @@ describe('todo test suite', () => {
     
 
     // Write all your test cases here that corresponds to software requirements
+    test("add_todo", () => {
```

Comment:
a better comment for explaining the function is implementing would be easier to understand